assets/js/atomic/blocks/product-elements/price/index.ts

```diff
@@ -1,22 +1,27 @@
 /**
  * External dependencies
  */
-import { isFeaturePluginBuild } from '@woocommerce/block-settings';
 import { registerBlockType } from '@wordpress/blocks';
+import type { BlockConfiguration } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
 import sharedConfig from '../shared/config';
 import edit from './edit';
 import attributes from './attributes';
+import { supports } from './supports';
 import {
 	BLOCK_TITLE as title,
 	BLOCK_ICON as icon,
 	BLOCK_DESCRIPTION as description,
 } from './constants';
 
-const blockConfig = {
+type CustomBlockConfiguration = BlockConfiguration & {
+	ancestor: string[];
+};
```

Comment:
I've added this part to prevent a TS error related to:

```ts
ancestor: [
  '@woocommerce/all-products',
  '@woocommerce/single-product',
  'core/post-template',
],
```

Reply:
I think we will need this for other atomic blocks too. I am curious why `ancestor` property isn't there in core `BlockConfiguration` type? 🤷🏻‍♂️
Tagging @gigitux, in case he knows about it 🙂

Reply:
> I think we will need this for other atomic blocks too. I am curious why ancestor property isn't there in core BlockConfiguration type? 🤷🏻‍♂️

I guess because no one has added it yet. That said, take a look at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63292. 😁